hs/src/Reach/APICut.hs

```diff
@@ -98,6 +98,7 @@ instance Contains DLStmt where
     DL_Only _ (Right _) t -> has q t
     DL_Only {} -> False
     DL_MapReduce {} -> False
+    DL_setApiDetails {} -> False
```

Comment:
Adding statements is a major pain. Why is this not a new expression evaluated for effect? I find that easier to deal with because most of the passes ignore them